playbooks/build-foreman-image.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: localhost
+  gather_facts: no
+  vars:
+    image: foreman
+  vars_files:
+    - vars/build-image-vars.yml
+  roles:
+    - role: create-workspace
+    - role: foreman-dockerfile
```

Comment:
Some bike shedding, to focus the role on that its about building a container image rather than the Dockerfile itself, you could name this:

 * foreman-container-image
 * foreman
 * foreman-image

Reply:
I renamed roles to foreman-image and foreman-proxy-image.